src/components/Tabs/Tabs.tsx

```diff
@@ -97,19 +61,9 @@ export interface Props {
  */
 export const Tabs = ({
   activeIndex = 0,
-  'aria-labelledby': ariaLabelledBy,
   children,
   className,
-  hideLegend,
-  id,
   onChange,
-  overflow,
-  radioLegend,
-  radioLegendPosition,
-  required,
-  size,
-  tabsOnClick,
-  variant,
```

Comment:
Yikes, all these props aren't used?

Reply:
This is one of those components that came as a template from Big Medium and was updated to work for our needs. I'm guessing these were all very useful in one or more of their previous projects.

Reply:
Yea, I pretty much assume any oddities are BM's fault.